lametro/models.py

```diff
@@ -141,21 +142,9 @@ def inferred_status(self):
         return self._status(description)
 
     def _status(self, description):
-        if description:
-            if description.upper() in ['APPROVED', 'APPROVED AS AMENDED', 'APPROVED THE CONSENT CALENDAR']:
-                return 'Approved'
-            elif description.upper() in ['ADOPTED', 'ADOPTED AS AMENDED']:
-                return 'Adopted'
-            elif description.upper() in ['RECOMMENDED FOR APPROVAL', 'RECOMMENDED FOR APPROVAL AS AMENDED', 'REFERRED', 'FORWARDED DUE TO ABSENCES AND CONFLICTS', 'FORWARDED WITHOUT RECOMMENDATION', 'NO ACTION TAKEN', 'NOT DISCUSSED']:
-                return 'Active'
-            elif description.upper() in ['RECEIVED', 'RECEIVED AND FILED']:
-                return 'Received'
-            elif description.upper() == 'FAILED':
-                return 'Failed'
-            elif description.upper() == 'WITHDRAWN':
-              return 'Withdrawn'
-        else:
-            return None
+        if description and description.upper() in BILL_STATUS_DESCRIPTIONS.items():
```

Comment:
`dict.items()` is an iterable of the key-value pairs in the dictionary, so I'm not sure this will work as you expect. You can do `key in dict.keys()` or `dict.get(key)`. TIL `key in dict.keys()` is faster!

Reply:
I did test this in a python shell with a simplified dictionary, but using `keys()` is more readable, I think. made the change, thanks!